alma/client.py

```diff
@@ -102,14 +162,12 @@ def __init__(
 
         if options["mode"] not in (ApiModes.LIVE, ApiModes.TEST):
             raise ValueError(
-                "`mode` option must be one of ({LIVE}, {TEST})".format(
-                    LIVE=ApiModes.LIVE.value, TEST=ApiModes.TEST.value
-                )
+                f"`mode` option must be one of ({ApiModes.LIVE.value}, {ApiModes.TEST.value})"
             )
 
         self.context = Context(options)
         self.init_user_agent()
-        self._endpoints = {}  # type: ignore
+        self._endpoints = {}  # type: Dict[str, endpoints.Base]
```

Comment:
If we drop 3.5 support then we should be able to do this:

```suggestion
        self._endpoints: Dict[str, endpoints.Base] = {}
```